.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,7 +9,6 @@ Before we can approve this PR for merge, please make sure that **all** the follo
 - [ ] Added **one** label out of the `L-` group to this PR
```

Comment:
`Linked to github issue ..` needs to be changed to be about zenhub connected issue.

Reply:
good point, updated it